Userland/Libraries/LibCore/Stream.cpp

```diff
@@ -153,16 +153,29 @@ bool File::exists(StringView filename)
     return !Core::System::stat(filename).is_error();
 }
 
+ErrorOr<NonnullOwnPtr<File>> File::standard_input()
+{
+    return File::adopt_fd(STDIN_FILENO, OpenMode::Read);
+}
+ErrorOr<NonnullOwnPtr<File>> File::standard_output()
+{
+    return File::adopt_fd(STDOUT_FILENO, OpenMode::Write);
+}
+ErrorOr<NonnullOwnPtr<File>> File::standard_error()
+{
+    return File::adopt_fd(STDERR_FILENO, OpenMode::Write);
+}
+
 ErrorOr<NonnullOwnPtr<File>> File::open_file_or_standard_stream(StringView filename, OpenMode mode)
 {
     if (!filename.is_empty() && filename != "-"sv)
         return File::open(filename, mode);
```

Comment:
I missed the creation of this API. Is this for convenience purposes for command line arguments? I'm not really a fan of it but I guess it's fine.

Reply:
It was added in #14674, and it's indeed a convenience helper for command line arguments.